rmw_connextdds_common/include/rmw_connextdds/rmw_impl.hpp

```diff
@@ -67,6 +67,14 @@ bool rmw_connextdds_find_string_in_list(
 DDS_Duration_t rmw_connextdds_duration_from_ros_time(
   const rmw_time_t * const ros_time);
 
+/**
+ * This function returns `NULL` when either the node handle is `NULL` or when the
+ * node handle is from a different rmw implementation.
+ *
+ * \return rmw_ret_t non `NULL` value if successful, otherwise `NULL`
```

Comment:
this comment seems wrong (the function does not return NULL and it does not take a node handle as argument)

Reply:
Ah whoops. That is left over from the actual implementation. I will fix.